src/components/Button/Button.js

```diff
@@ -39,12 +45,15 @@ const styles = StyleSheet.create({
     alignItems: 'center'
   },
   buttonText: {
-    color: COLORS.white,
+    color: COLORS.white2,
     fontSize: 18,
     fontFamily: fonts.inter[600]
   },
+  buttonTextDisabled: {
+    color: COLORS.gray300
+  },
   disabledbutton: {
-    backgroundColor: COLORS.gray300,
+    backgroundColor: COLORS.gray200,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's a minor typo in the style name `disabledbutton`. It should be camelCase for consistency with other style names. 

```diff
-   disabledbutton: {
+   disabledButton: {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->